src/pages/academy/index.tsx

```diff
@@ -91,7 +93,9 @@ const DerivBlog = ({ data }: DerivBlogProps) => {
     let recent_data = data.directus.recent
     let featured_data = data.directus.featured
     let homepage_banner_data = data.directus.homepage_banners
-    let non_featured_video_list_data = data.directus.videos
+    let non_featured_video_list_data = data.directus.videos.filter((item) => {
+        return !is_p2p_allowed_country ? !item.video_title.includes('Deriv P2P') : item
```

Comment:
Let's get this text 'Deriv P2P' from a constant

Reply:
okay